### DISCUSSION TITLE
A bit-off topic - has anyone tried Sonoma Photos yet?

### DISCUSSION THREAD

**Discussion by cfc62** [RhetTbull/osxphotos#1237] [General]
As I've been bit before in the last few MacOS releases by regressions, I'm more gun-shy now. 

As an example, there was a terrible regression in the middle of the Ventura releases where shares with custom disk icons just stopped working and left me high and dry for weeks. This wasn't during a beta, where it might be more understandable, but in the middle of the release cycle, and a clear regression which was next to inexcusable. 

In general, it would at least seem that the overall chatter on the internet is that Sonoma is usable. I see some issues around external drives not mounting, which likely has a specific trigger, and I'm wary of the changes put in enhanced screen sharing as I have 2 headless servers and Apple finally got screen sharing to work wonderfully with them, including the boot process. 

In specific though, any experiences around Photos in Sonoma, positive or negative?

**Comment by RhetTbull** [comment]
I don't typically upgrade until at least the first `.1` release.  I have tested osxphotos on the Sonoma dev preview and it works. I have not yet tried the final release.  osxphotos does not yet support pet faces (a new feature) but I expect existing features will work.

**Reply by cfc62** [reply]
I don't know what happened to Apple's QA/QC but it certainly hasn't gotten better... 

In the latest version of Photos for Ventura Version 8.0 (560.0.110) in 13.6, there's a RAW display bug that's made it into the shipped product, sounds like it's in the Sonoma version as well. 

It's easily found and is easily replicable. I am a bit worried that Apple is leaning a bit too hard on the public betas to find issues. I shouldn't have to file a bug report on something like this. And I'll add this regression came in what should have been the final release of Ventura. How does that even happen?

**Reply by RhetTbull** [reply]
> . I am a bit worried that Apple is leaning a bit too hard on the public betas to find issues

Yeah, I agree. I'm surprised by some of the things they've shipped. They also changed the release cadence to new macOS every 12 months instead of 18-24 months. I'm sure that puts a lot of pressure on dev teams to just ship it.
